### PR TITLE
fix: position of badges

### LINKED ISSUE #4649: /booking: move all badges below the time
all badges:
- [ ] unconfirmed
- [ ] unpaid
- [ ] the team badge

CleanShot 2022-09-21 at 15 16 12@2x

team badge: 
CleanShot 2022-09-21 at 15 18 43@2x

like this:

CleanShot 2022-09-21 at 15 17 20@2x

### PR DESCRIPTION
**Pull request by Udit-takkar**
fixes: #4649 

Screenshot 2022-09-22 at 2 00 22 AM

### CODE REVIEW

**Comment by hariombalhara** [comment]
Screenshot 2022-09-22 at 10 06 38 AM
I think we should show the label in mobile mode as well. cc @Jaibles for the design

**Comment by Jaibles** [comment]
We should definitely show the label on mobile. As well as the date & time.

CleanShot 2022-09-22 at 09 12 30@2x

 It would be great to also test this with recurring bookings.

**Comment by Udit-takkar** [comment]
ok

**Comment by Udit-takkar** [comment]
Screenshot 2022-09-24 at 6 11 28 PM

should i also change the button(Edit booking) like you have done here ?

**Comment by Udit-takkar** [comment]
Screenshot 2022-09-24 at 6 24 11 PM

build this for mobile view

File: apps/web/components/booking/BookingListItem.tsx [hunk 1]
```diff
@@ -262,73 +261,92 @@ function BookingListItem(booking: BookingItemProps) {
...
             <div className="text-sm leading-6 text-gray-900">{startTime}</div>
             <div className="text-sm text-gray-500">
               {dayjs(booking.startTime).format(user && user.timeFormat === 12 ? "h:mma" : "HH:mm")} -{" "}
               {dayjs(booking.endTime).format(user && user.timeFormat === 12 ? "h:mma" : "HH:mm")}
             </div>
+
+            {isPending && (
+              <>
+                <Badge variant="orange">{t("unconfirmed")}</Badge>
+                <br />
```
Comment (PeerRich) [inline apps/web/components/booking/BookingListItem.tsx:276]: `<br>` is not a good way to change layout. just wrap in a div and give mb

**Comment by PeerRich** [comment]
@Udit-takkar left a comment re: not using` <br>` for layouting

**Comment by Udit-takkar** [comment]
done

**Review by PeerRich — APPROVED** [review]
great stuff! thank you @Udit-takkar

**Comment by zomars** [comment]
Just a FYI @Udit-takkar @PeerRich this PR is what currently breaking our E2E tests. We're introducing duplicate markdown and displaying it with breakpoints and Playwright doesn't like that.

File: apps/web/components/booking/BookingListItem.tsx [hunk 2]
```diff
@@ -350,9 +365,20 @@ function BookingListItem(booking: BookingItemProps) {
...
+              {isUpcoming && !isCancelled ? (
+                <>
+                  {isPending && user?.id === booking.user?.id && <TableActions actions={pendingActions} />}
+                  {isConfirmed && <TableActions actions={bookedActions} />}
+                  {isRejected && <div className="text-sm text-gray-500">{t("rejected")}</div>}
+                </>
+              ) : null}
+              {isPast && isPending && !isConfirmed ? <TableActions actions={bookedActions} /> : null}
+            </div>
+          </div>
```
Comment (zomars) [inline apps/web/components/booking/BookingListItem.tsx:379]: This code block specifically is what is breaking our tests:

image

Comment (zomars) [reply apps/web/components/booking/BookingListItem.tsx:379]: cc @emrysal for context

**Pull request merged**